editor/editor_autoload_settings.cpp

```diff
@@ -823,11 +824,23 @@ EditorAutoloadSettings::EditorAutoloadSettings() {
 	l->set_text(TTR("Path:"));
 	hbc->add_child(l);
 
-	autoload_add_path = memnew(EditorLineEditFileChooser);
+	autoload_add_path = memnew(HBoxContainer);
```

Comment:
`autoload_add_path` can be local, no need to store the reference on class.

Reply:
No. We can't make it local. `autoload_add_path` is used in other functions as well.

Reply:
What other functions? You changed everything to direct references for `line_edit` and `file_dialog`, and that is what it was used for before, to retrieve those.

Reply:
My bad. I changed them later. Will fix it.

Reply:
I removed it completely and added it to hbc itself.